client/src/components/Display/MoneyDisplay/BottomContent/Footer/ExchangeRate/ExchangeRate.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+import s from './ExchangeRate.module.less';
+import { parseMoney } from '@/utils';
+
+interface Props {
+  exchangeName: baseExchangeName;
+  exchangeRate: number;
+}
+
+type baseExchangeValue = '€' | '$' | '£';
+type baseExchangeName = 'eur' | 'usd' | 'gbp';
+
+export const ExchangeRate: FC<Props> = ({ exchangeRate, exchangeName }) => {
+  const exchangeRateValues: Record<baseExchangeName, baseExchangeValue> = {
+    usd: '$',
+    eur: '€',
+    gbp: '£',
+  };
```

Comment:
Can we keep it outside of component? Now this object is created on every re-render

Reply:
Fixed